docs-sources/includes/_Archivist.md

```diff
@@ -496,3 +524,74 @@ to only data which matches the userId.
 
 We then use the acl function to only allow users and tests access to
 the `get` API, but full access to CMS users and administrators.
+
+## Migrations
+
+MAGE supports database migration scripts similar to [Ruby on Rails 2.1](http://api.rubyonrails.org/classes/ActiveRecord/Migration.html)
+which are aligned with your `package.json` version and exposed via the `archivist:migrate`
```

Comment:
As long as they're smaller or equal, they work. Yeah, this was very much by design, but it actually creates confusion I noticed. For now, it's what it is, but this is definitely worth re-evaluating.